pkgs/development/python-modules/attrs-strict/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  python3
+}:
+
+buildPythonPackage rec {
+  pname = "attrs_strict";
+  version = "1.0.1";
+  disabled = pythonOlder "3.7";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-5wSGNiAUbF8qi2Ac1FdNFIkT2yb8Bjb5Qf5CEuQl6v4=";
+  };
+
+  build-system = [
+    python3.pkgs.setuptools
+    python3.pkgs.setuptools-scm
+  ];
+
+  dependencies = with python3.pkgs; [
+    attrs
+  ];
+
+  pythonImportsCheck = [ "attrs_strict" ];
+
+  meta = {
+    description = "Python package which contains runtime validation for attrs data classes based on the types existing in the typing module";
+    homepage = "https://github.com/bloomberg/attrs-strict";
+    changelog = "https://github.com/bloomberg/attrs-strict/releases/tag/${version}";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ ];
```

Comment:
If you want to have them, you'll have to maintain them.

Reply:
Fixed